lib/templates/decodable/README.md

```diff
@@ -34,3 +34,6 @@ extension Shop: Decodable {
   }
 }
 ```
+# Specific JSON tasks
```

Comment:
Depending on your markdown interpreter, some Markdown renderers are a little strict and expect an empty line after section titles, otherwise they won't render it as titles. So better add empty newlines after those ;)